### ISSUE TITLE
Shorten the package name

### ISSUE THREAD

**Issue by tomalec** [woocommerce/grow#7]
_This is kind of a follow up from https://github.com/woocommerce/grow/pull/6#discussion_r879919622_

## User story

It's rather a cosmetic :nail_care:  problem, but I think it'd be nice to settle on good names, as later it could be hard to ever them.

As a user of Grow tools, I'd like to write less code/configs/ etc. and avoid repetition.
Also when it comes to the package name.

### Is your feature request related to a problem? 

Currently, we use a format of `woocommerce-grow-{package-name}` for our tools here. (As it was introduced in [the very first package](https://github.com/woocommerce/grow/pull/1/files#diff-38c8e345663d6745c228f9a299ff50656ddad71bdfe911b2745c69610cc84cbdR2))

It has a set of benefits:
- states the full product name, probably good for recognition
- states the team name, so it's distinct from Woo-wide tools
- does not use shorthand which makes it kind of disambiguous, and readable out loud 1:1 in plain English

But I have a doubt, whether it's not too long. Especially, when the package name adds up, and the name is used not only in `package.json`'s `devDependencies`, but also in some other files, like configs, like:

- Some tags in .md files [`<woo-tracking-jsdoc></woo-tracking-jsdoc>` ](https://github.com/woocommerce/google-listings-and-ads/blob/develop/src/Tracking/README.md?plain=1#L75)
- Commands in [`package.json`'s `scripts`](https://github.com/woocommerce/google-listings-and-ads/blob/develop/package.json#L38-L40) which to be fully consistent would be renamed to:
   ```
   "storybook": "woocommerce-grow-storybook",
   "storybook:build": "woocommerce-grow-storybook build",
   "storybook:deploy": "woocommerce-grow-storybook deploy",
   ```
- Config files as mentioned in https://github.com/woocommerce/grow/pull/6#discussion_r881599737 (however having that config file is a matter of another discussion)
  

### Describe the solution you'd like 

I see a few ways to go:

1. Don't make any hard rule and decide ad hoc for every case separately, the package name, the config references, the binaries' names, element names etc.
2. Make `woocommerce-grow-{package-name}` for every package reference
3. Use something shorter, like `woo-grow-{package-name}`
   1. Use it everywhere
   2. Use a long name for packages, a short name for any references

For `woo-grow-` I could list a few benefits
- shorter
- is well-coined shorthand
- Does not necessarily reference the WooCommerce product, but rather the Woo division. And we may have some tools not just for WooCommerce per see, but for some other needs. I cannot think of a good example now - maybe something like `woo-grow-ping-porter` CLI/slack command. Also Grow is not a part of WooCommerce plugin, but rather Woo as an organization.
- States a clear distinction between those packages and `woocommerce` packages. - "Is `woocommerce-grow-up` a WooCommerce's "grow-up" package, or WooCommerce Grow's `up` package?"

**Comment by eason9487** [comment]
Since both `github-actions` and JS packages now follow the same release process to shorten their reference names, I think this issue can be considered resolved.
